src/components/Backdrop/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import './style.scss';
+import scenery from 'assets/common_backdrop/scenery.svg';
+import sceneryMobile from 'assets/common_backdrop/scenery_mobile.svg';
+import background from 'assets/common_backdrop/background.png';
+import topSign from 'assets/common_backdrop/top-sign.svg';
+import NavBar from 'components/NavBar';
+
+const signRotations = ['9.03deg', '0deg', '-8deg', '0deg'];
+
+export default function Backdrop(props) {
+  return (
+    <div className="backdrop">
```

Comment:
It's (arguably) good style to capitalize the className here, i.e. `Backdrop`, because it's the name of a class. Feel free to disagree